<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string>imageLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>uploadButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cameraButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>420</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>cameraButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="encryptButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>420</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>encryptButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
